### PR TITLE
Ad block - add preview

### PR DESCRIPTION
**Pull request by MichaelArestad**
Master issue: #13510

@joanrho This shows the same preview we show in the editor. It's not as nice as the image preview we show, but this is a start for sure and I haven't had great luck implementing the image preview.

#### Changes proposed in this Pull Request:

* Added an example preview to the block in the Inspector.

#### Before
image

#### After
image

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Adding support for a new feature

#### Testing instructions:

* Get a Jetpack Premium Plan or higher
* Go to Jetpack > Settings > Traffic and enable Ads
* Create a new post
* Click the + to open the top Inserter
* Hover over Ad block

#### Proposed changelog entry for your changes:

* None needed

### CODE REVIEW

**Comment by simison** [comment]
cc @dbspringer @rclations in case you have ideas!

**Comment by joanrho** [comment]
Nice work, @MichaelArestad. Just tested and everything looks/works as expected. This is a big improvement from no preview if we can't insert an image of a sample ad (like in our design mock).

**Comment by dbspringer** [comment]
I'm no designer, but this seems fine to me--it's pretty accurate to what the block looks like in the editor. Are there some other examples I can take a look at as an idea of what else is possible?

As long as we have the correct dimensions for the unit and it behaves accurately with formatting, I'm not super convinced a mock ad is as important.

**Comment by MichaelArestad** [comment]
@dbspringer Generally, they show a preview of what the block will look like on the front end (which is what they _should_ look like in the editor, but for various reasons can't always do. In this case, the only thing really sticking out to me is the toggle. That control should probably only be visible when the block is selected in the editor.

**Comment by dbspringer** [comment]
We can feel free to grab one of our house ads in `themes/a8c/pubpress/houseads/` if we want to use one of them as a stock "that is what an ad looks like" preview.

**Comment by MichaelArestad** [comment]
@dbspringer That would be ideal.

**Comment by MichaelArestad** [comment]
I think I can make this happen after some tests. On it.

**Comment by MichaelArestad** [comment]
Check it out!
### Preview:
image


### When selected:
image

### When unselected:
image

### Previews for all formats:
image

CC @joanrho

**Comment by MichaelArestad** [comment]
Note: the toggle alignment is strange. Partially because of the theme's styles. That needs to be fixed in a separate PR.

**Comment by joanrho** [comment]
LGTM. Thanks!

File: extensions/blocks/wordads/edit.js [hunk 1]
```diff
@@ -4,27 +4,53 @@
...
+		const adControls = (
+			<ToggleControl
+				className="jetpack-wordads__mobile-visibility"
+				checked={ Boolean( hideMobile ) }
+				label={ __( 'Hide ad on mobile views', 'jetpack' ) }
+				onChange={ this.handleHideMobileChange }
+			/>
+		);
+		function getExampleAd( formatting ) {
+			switch ( formatting ) {
```
Comment (dereksmart) [inline extensions/blocks/wordads/edit.js]: @MichaelArestad should we add a `default` case for this switch? Not sure how likely/possible it is for there to be a value that is not covered in the current cases

Comment (simison) [reply extensions/blocks/wordads/edit.js]: Feels like that could cover a bug somewhere in a way that it's hard to notice?

Comment (MichaelArestad) [reply extensions/blocks/wordads/edit.js]: TBH, I couldn't remember how to write the default (though I think it's just `default:`) and I defer to y'all.

Comment (dereksmart) [reply extensions/blocks/wordads/edit.js]: If deferring to me, then yes will you please add a default value?

Comment (MichaelArestad) [reply extensions/blocks/wordads/edit.js]: @dereksmart On it.

**Comment by MichaelArestad** [comment]
Updated.

@joanrho @jeherve updated the images to look like this:
image

@dereksmart added a default in the switch case

**Review by kraftbj — APPROVED** [review]

**Pull request merged**